lib/routes/utils/request_helper.js

```diff
@@ -9,9 +9,52 @@
  * @returns {boolean}
  */
 function wantsKeys (request) {
-  return request.query.keys === 'true'
+  return request.query.keys === 'true' || request.query.keys === true
```

Comment:
This was a self inflicted bug introduced when using the mocked requests :(, fixed.